.changeset/tame-lemons-play.md

```diff
@@ -0,0 +1,194 @@
+---
+"@latticexyz/cli": patch
+"@latticexyz/common": major
+"@latticexyz/create-mud": major
+"@latticexyz/recs": patch
+"@latticexyz/store-indexer": patch
+---
+
+Templates and examples now use MUD's new sync packages, all built on top of [viem](https://viem.sh/). This greatly speeds up and stabilizes our networking code and improves types throughout.
+
+These new sync packages come with support for our `recs` package, including `encodeEntity` and `decodeEntity` utilities for composite keys.
+
+If you're using `store-cache` and `useRow`/`useRows`, you should wait to upgrade until we have a suitable replacement for those libraries. We're working on a [sql.js](https://github.com/sql-js/sql.js/)-powered sync module that will replace `store-cache`.
+
+## Migrate existing RECS apps to new sync packages
```

Comment:
a headline in the changeset might look weird in the changelog, how about changing this to just bold text?
```suggestion
**Migrate existing RECS apps to new sync packages**
```